0001-organization-of-ADRs.md

```diff
@@ -0,0 +1,67 @@
+# Initial plan for organization of ADR files
+
+* Status: proposed 
+* Deciders: @ZavenArra {list everyone involved in the decision using github handle} <!-- optional -->
+* Date: 2022-05-11 {YYYY-MM-DD when the decision was last updated} <!-- optional -->
+
+
+## Context and Problem Statement
+
+We have many decisions to make when building the treetracker platform, and we will use Architectural Design Recores (ADRs) to make more and more of these decisions. (https://adr.github.io/madr/)  ADRs could be organized in a few different ways - should all decisions to be tracked in one central ADR repository, or should they be split up by teams or project areas?  ADRs are assumed to be stored in a github repository or several repositories.
```

Comment:
Typo ADR should be read as Architectural Decision Records  and not Architectural Design Records